### ISSUE TITLE
C-u dired-narrow (universal prefix) should remove all matching items

### ISSUE THREAD

**Issue by novoid** [Fuco1/dired-hacks#157]
Hi,

I love `dired-narrow`, I use it on a daily basis.

However, sometimes, I'd like to have the opposite feature: hide everything that matches the pattern. Therefore, I propose a **feature-wish** where `C-u /` (universal prefix + `dired-narrow`) is removing all items that match the pattern.

**Comment by Fuco1** [comment]
Makes sense to me.  Do you want to take a shot at implementing this?  I can guide you if necessary.

**Comment by novoid** [comment]
Hi,
Well.... I'm not really into Elisp (unfortunately) but I might try. Although, I can't try before end of August or begin of September since I just noticed that my hotel WiFi is somewhat partially broken.
